scripts/convert_pipenv_to_requirements.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+#
+#    Copyright 2019 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+import argparse
+from os import path
+
+from requirementslib import Pipfile
```

Comment:
Did you add information that we need requirementslib anywhere?

Reply:
Please, merge your PR with documentation and I would love to do that.